### ISSUE TITLE
Can the ocr approach implemented here be used for converting dvb_subtitles to webvtt in live streaming application?

### ISSUE THREAD

**Issue by hariszukanovic** [softworkz/SubtitleFilteringDemos#3]

**Comment by softworkz** [comment]
Yes, when it uses segmented streaming like HLS. That's actually (one of) my primary use case(s).

**Comment by hariszukanovic** [comment]
I would like very much to test... would you kindly provide an example command for HLS output with subs?
How about Dash? Any chance that it works too? :)

**Comment by Neo2SHYAlien** [comment]
> Yes, when it uses segmented streaming like HLS. That's actually (one of) my primary use case(s).

@softworkz  Hi. Thank you for your hard work! Unfortunately I didn't find a way of packaging again the subs after ocr process

**Comment by cooliobr** [comment]
`TESSDATA_PREFIX=/usr/local/src/FFmpeg/tesseract/tessdata ./ffmpeg -hwaccel cuvid -deint 2 -fix_sub_duration -drop_second_field true -analyzeduration 1000000 -probesize 1000000 -i "udp://225.200.200.7:1234?fifo_size=100000" -filter_complex "[0:v:0]scale_npp=w=1280:h=720:interp_algo=lanczos,scale_npp=format=nv12,yadif_cuda=0:-1:1,format=cuda[v0out];[0:s:0]graphicsub2text=delay_when_no_duration=1[s01]" -map "[v0out]" -c:v h264_nvenc -gpu 0 -r 30 -g 15 -keyint_min 15 -muxdelay 0 -b:v 3500k -maxrate 4500k -bufsize 3000k -profile:v high -preset fast -map "0:a:1" -c:a:0 aac -b:a:0 256k -ar 48000 -ac 2 -metadata:s:a:0 language=por -map "0:a:0" -c:a:1 aac -b:a:1 256k -ar 48000 -ac 2 -metadata:s:a:1 language=eng -map "[s01]" -c:s webvtt -metadata:s:s:0 language=por -f hls -hls_time 0.5 -hls_list_size 4 -fflags nobuffer -flags low_delay -hls_flags append_list+omit_endlist+delete_segments -hls_segment_type mpegts -hls_segment_filename /output/canal_1/stream_%v-%d.ts -master_pl_name playlist.m3u8 -var_stream_map "a:0,agroup:aud_low,default:yes,language:POR,name:Portuguese a:1,agroup:aud_low,language:ENG,name:English v:0,agroup:aud_low,s:0,sgroup:subtitle,name:video" /output/canal_1/stream_%v.m3u8` this command is working fine, but is possible migrate to ffmpeg 7 ?
